apps/web/src/app/[locale]/(dashboard)/[wsId]/users/groups/[groupId]/schedule.tsx

```
@@ -0,0 +1,194 @@
+                      <div
+                        key={`${groupId}-${currentDate.toDateString()}-day-${idx}`}
+                        className="bg-foreground/5 text-foreground/40 dark:bg-foreground/10 flex flex-none cursor-default justify-center rounded border p-2 font-semibold transition duration-300 md:rounded-lg"
+                      >
+                        {day.getDate()}
+                      </div>
+                    );
+
+                  return (
+                    <div
+                      className={`flex flex-none cursor-default justify-center rounded border border-green-500/30 bg-green-500/10 p-2 font-semibold text-green-600 transition duration-300 md:rounded-lg dark:border-green-300/20 dark:bg-green-300/20 dark:text-green-300`}
+                    >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)